index.html

```diff
@@ -2349,6 +2349,94 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           </table>
         </div>
 
+        <p>
+          A <a>Thing Description</a> of a WoT producer may have multiple forms entries with, e.g., different 
+          protocol and/or content types declarations that a <a>Consumer</a> could possibly support. In that case 
+          the Consumer may choose any form entry that works (e.g., the protocol and content type is supported) for them. 
```

Comment:
should that be an assertion?

Reply:
Good point. I think that the second one is indeed an assertion